usaspending_api/awards/models.py

```diff
@@ -84,6 +84,17 @@ class Meta:
         db_table = 'financial_accounts_by_awards_transaction_obligations'
 
 
+class AwardManager(models.Manager):
+    def get_queryset(self):
+        q_kwargs = {
+            "type": "U",
+            "total_obligation__isnull": True,
```

Comment:
Thanks for clarifying. That all makes sense, but I can see how this might look arbitrary to someone not familiar with how all of this works. What do you think about putting some of your explanation into a comment?